README.md

```diff
@@ -40,7 +39,6 @@ delegates = ark.delegates().delegates()
 - [ ] add P2P API calls
 - [ ] fix Peer().peer()
 - [ ] fix both UnconfirmedTransactions functions in Transaction()
-- [ ] Transport function is wonky when installed
```

Comment:
don't remove it, just do `[x] Transport function is wonky when installed`

Reply:
Added back, thanks for the suggestion